viewer/packages/utilities/src/customObject/CustomObjectIntersection.ts

```diff
@@ -0,0 +1,24 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+
+import { Vector3, Box3 } from 'three';
+import { CustomObject } from './CustomObject';
+
+/**
+ * This class is used as output intersecting custom objects
+ * @beta
+ */
+export class CustomObjectIntersection {
```

Comment:
I wonder if this should be just a type rather than a class :thinking: If we want to combine it in the global `Intersection` type, it should be defined similarily to how `CadIntersection` and `PointCloudIntersection` are defined, in `packages/api/src/public/migration/types.ts`. I don't see the benefit of letting this be a class